src/main/java/org/gridsuite/modification/server/modifications/VscCreation.java

```diff
@@ -0,0 +1,316 @@
+
+        List<Report> setPointsReports = new ArrayList<>();
+        Reporter setPointsReporter = subReporter.createSubReporter("vscSetPoints", "Setpoints");
+        setPointsReports.add(ModificationUtils.getInstance().buildCreationReport(modificationInfos.getConvertersMode(), "Converters mode"));
+        setPointsReports.add(ModificationUtils.getInstance().buildCreationReport(modificationInfos.getActivePower(), "Active power"));
+        setPointsReports.add(ModificationUtils.getInstance().buildCreationReport(modificationInfos.getMaximumActivePower(), "Pmax"));
+        ModificationUtils.getInstance().reportModifications(setPointsReporter, setPointsReports, "vscSetPoints", "Setpoints");
+
+        List<Report> angleDroopActivePowerControlReports = new ArrayList<>();
+        angleDroopActivePowerControlReports.add(Report.builder().withKey("angleDroopActivePowerControl")
+                .withDefaultMessage(modificationInfos.getAngleDroopActivePowerControl() ? "enabled" : "disabled")
+                        .withSeverity(TypedValue.INFO_SEVERITY)
+                .build());
+        if (modificationInfos.getP0() != null) {
+            angleDroopActivePowerControlReports.add(ModificationUtils.getInstance().buildCreationReport(modificationInfos.getP0(), "P0"));
+        }
+
+        if (modificationInfos.getDroop() != null) {
+            angleDroopActivePowerControlReports.add(ModificationUtils.getInstance().buildCreationReport(modificationInfos.getDroop(), "Droop"));
+        }
+
+        ModificationUtils.getInstance().reportModifications(setPointsReporter, angleDroopActivePowerControlReports, "vscAngleDroop", "Angle droop active power control");
+    }
+
+    private VscConverterStation createConvertStation(Network network,
+                                                     ConverterStationCreationInfos converterStationCreationInfos,
+                                                     Reporter subReporter,
+                                                     String logFieldName) {
+        Reporter converterStationReporter = subReporter.createSubReporter("converterStationCreated", logFieldName);
+        List<Report> converterStationReports = new ArrayList<>();
+        VoltageLevel voltageLevel = ModificationUtils.getInstance().getVoltageLevel(network, converterStationCreationInfos.getVoltageLevelId());
+        VscConverterStation vscConverterStation = voltageLevel.getTopologyKind() == TopologyKind.NODE_BREAKER ?
+                createConvertStationInNodeBreaker(network, voltageLevel, converterStationCreationInfos, converterStationReports, converterStationReporter) :
+                createConvertStationInBusBreaker(voltageLevel, converterStationCreationInfos, converterStationReports, converterStationReporter);
+
+        converterStationReporter.report(Report.builder()
+                .withKey("converterStationCreated")
```

Comment:
you cannot use the same key here for converter station 1 and converter station 2